bin/update_pythons.py

```diff
@@ -196,13 +196,19 @@ def __init__(self) -> None:
                 uri = int(release["resource_uri"].rstrip("/").split("/")[-1])
                 self.versions_dict[version] = uri
 
-    def update_version_macos(self, identifier: str, spec: Specifier) -> ConfigMacOS | None:
-        file_idents = ("macos11.pkg", "macosx10.9.pkg", "macosx10.6.pkg")
+    def update_version_macos(
+        self, identifier: str, version: Version, spec: Specifier
+    ) -> ConfigMacOS | None:
         sorted_versions = sorted(v for v in self.versions_dict if spec.contains(v))
 
-        for version in reversed(sorted_versions):
+        if version <= Version("3.8.9999"):
+            file_idents = ("macosx10.9.pkg",)
+        else:
+            file_idents = ("macos11.pkg", "macosx10.9.pkg")
```

Comment:
Should we just drop the tuple here and just go ahead and define `file_ident` once and for all ?

Reply:
Nevermind, I went this way just to silence the MyPy issue - I guess I could have solved differently.